CONTRIBUTING.md

```diff
@@ -1,6 +1,11 @@
 
 ## Acceptable Contributions
 
+-Do you want to become a contributor for all solutions of problems on www.binarysearch.com?
```

Comment:
Bro change "for all solutions of ........ binarysearch.com?"  to  "the (awesome) repository"

Reply:
Hop bro